msteams-platform/m365-apps/extend-m365-teams-message-extension.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Extend a Teams message extension across Microsoft 365
-description: In this article, you'll learn how to update search-based Teams message extension to run in Outlook by updating app manifest, adding an outlook channel, and sideloading updated app.
-ms.date: 05/24/2022
+description: Here's how to update your search-based message extension to run in Outlook, in addition to Microsoft Teams.
```

Comment:
As per standards, we should either use "In this learn module", "In this module", "Learn how to", "With this learning path. "Here's how" is not as per the guidelines.